manifests/0000_20_kube-apiserver-operator_00_cr-anyuid.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
```

Comment:
i dont really like this term as i would anticipate hypershift-installer working on aws and not needing new manifests.